util/create-email.ts

```diff
@@ -0,0 +1,46 @@
+import { v4 as uuid } from 'uuid';
+import { sparqlEscapeDateTime, sparqlEscapeString } from './mu';
+import { updateSudo } from '@lblod/mu-auth-sudo';
+
+const EMAIL_FROM_MANDATARIS_EFFECTIEF =
+  process.env.EMAIL_FROM_MANDATARIS_EFFECTIEF;
+export const SEND_EMAILS =
+  process.env.SEND_EMAIL_FOR_MANDATARIS_EFFECTIEF ?? false;
+
+if (SEND_EMAILS && !EMAIL_FROM_MANDATARIS_EFFECTIEF) {
+  throw 'Please set the email adres to the value set in the LMB app EMAIL_FROM_MANDATARIS_EFFECTIEF must equal to EMAIL_ADDRESS';
+}
+console.log(
+  `EMAIL_FROM_MANDATARIS_EFFECTIEF SET TO: ${EMAIL_FROM_MANDATARIS_EFFECTIEF}`,
+);
+console.log(`SEND_EMAIL_FOR_MANDATARIS_EFFECTIEF SET TO: ${SEND_EMAILS}`);
+
+export async function sendMailTo(emailTo: string, mandataris) {
+  const from = sparqlEscapeString(EMAIL_FROM_MANDATARIS_EFFECTIEF as string);
+  const to = sparqlEscapeString(emailTo);
+  const description = `De publicatie status van ${mandataris.name} met mandaat ${mandataris.mandate} en uri ${mandataris.uri} staat al 10 dagen of meer op effectief zonder dat er een besluit is toegevoegd. Gelieve deze mandataris manueel te bekrachtigen en een besluit toe te voegen of publiceer het besluit van de installatievergadering via een notuleringspakket.`;
```

Comment:
Improved the mail a little bit, but could still be better, but I would keep it like this, until Tim asks otherwise.